packages/puppeteer-core/src/api/HTTPResponse.ts

```diff
@@ -67,7 +67,9 @@ export class HTTPResponse {
    * True if the response was successful (status in the range 200-299).
    */
   ok(): boolean {
-    throw new Error('Not implemented');
+    // TODO: document === 0 case?
```

Comment:
is there a left-over common implementation?

Reply:
This implementation is common for both  BiDi and CDP `ok()`. Comment is from the CDP file just moved to API.